pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -1,109 +1,727 @@
+    meta = attrs.meta // {
+      license = lib.licenses.mitOpenGroup;
+      mainProgram = "bdftopcf";
+    };
   });
 
-  bitmap = addMainProgram super.bitmap { };
+  bitmap = prev.bitmap.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      license = lib.licenses.mitOpenGroup;
+      mainProgram = "bitmap";
+    };
+  });
 
-  editres = super.editres.overrideAttrs (attrs: {
+  editres = prev.editres.overrideAttrs (attrs: {
     hardeningDisable = [ "format" ];
-    meta = attrs.meta // { mainProgram = "editres"; };
+    meta = attrs.meta // {
+      license = lib.licenses.mitOpenGroup;
+      mainProgram = "editres";
+    };
   });
 
-  fontmiscmisc = super.fontmiscmisc.overrideAttrs (attrs: {
-    postInstall =
-      ''
-        ALIASFILE=${xorg.fontalias}/share/fonts/X11/misc/fonts.alias
-        test -f $ALIASFILE
-        cp $ALIASFILE $out/lib/X11/fonts/misc/fonts.alias
-      '';
+  encodings = prev.encodings.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      license = lib.licenses.publicDomain;
+    };
+  });
+
+  fontadobe100dpi = prev.fontadobe100dpi.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      # HPND-sell-variant, but with Adobe trademark and no disclaimer
+      license = lib.licenses.free;
+    };
+  });
+
+  fontadobe75dpi = prev.fontadobe75dpi.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      # same as fontadobe100dpi
+      license = lib.licenses.free;
+    };
+  });
+
+  fontadobeutopia100dpi = prev.fontadobeutopia100dpi.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      license = lib.licenses.adobeUtopia;
+    };
+  });
+
+  fontadobeutopia75dpi = prev.fontadobeutopia75dpi.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      license = lib.licenses.adobeUtopia;
+    };
+  });
+
+  fontadobeutopiatype1 = prev.fontadobeutopiatype1.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      license = lib.licenses.adobeUtopia;
+    };
+  });
+
+  fontalias = prev.fontalias.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      license = lib.licenses.cronyx;
+    };
+  });
+
+  fontarabicmisc = prev.fontarabicmisc.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      license = lib.licenses.mit;
+    };
+  });
+
+  fontbh75dpi = prev.fontbh75dpi.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      license = lib.licenses.lucidaBitmapFonts;
+    };
+  });
+
+  fontbh100dpi = prev.fontbh100dpi.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      license = lib.licenses.lucidaBitmapFonts;
+    };
+  });
+
+  fontbhlucidatypewriter75dpi = prev.fontbhlucidatypewriter75dpi.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      # only includes a copyright
+      license = lib.licenses.unfree;
+    };
+  });
+
+  fontbhlucidatypewriter100dpi = prev.fontbhlucidatypewriter100dpi.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      # only includes a copyright
+      license = lib.licenses.unfree;
+    };
+  });
```

Comment:
These have previously been covered here: https://github.com/NixOS/nixpkgs/issues/99014. See especially https://github.com/NixOS/nixpkgs/issues/99014#issuecomment-827859023. It looks like it may be a casualty of repository splitting. I think these are as fine as the other Lucida fonts, and Debian and Fedora have them marked accordingly.